### PR TITLE
Visitor can read specific article

### PR DESCRIPTION
**Pull request by mauroavellaneda**
[PT](https://www.pivotaltracker.com/story/show/175137693)
## This PR contains:
- Acceptance test
- Get request for a particular article when clicking on the article card.
- SingleArticle component
- Styling with Semantic UI for single article page. 
- Refactoring of DisplayArticles  into a functional component with the use of hooks. 
- Other refactoring while addressing feedback from previous PR.

### CODE REVIEW

**Comment by mauroavellaneda** [comment]
Screenshot 2020-10-09 at 12 01 46

**Review by FacundoOs — APPROVED** [review]

**Review by SebastianN97 — APPROVED** [review]

File: cypress/fixtures/articles_index.json [hunk 1]
```diff
@@ -2,23 +2,27 @@
   "articles": [
     {
       "id": 1,
-      "title": "Manchester City loves e-scooters",
-      "lead": "Lorem ipsum dolor sit amet consectetur adipisicing elit. Aut odit eius facere tempora expedita, ratione itaque quidem sequi totam nulla? Eaque possimus sit a officia."
+      "title": "Ny studie: Möjligt att utplåna fattigdomen och samtidigt rädda klimatet",
+      "lead": "Den ”omöjliga ekvationen” går att lösa.",
+      "content": "Det är fullt möjligt att rädda jordens klimat och utplåna fattigdomen på samma gång, hävdar brittiska forskare. Och detta utan att kasta oss tillbaka till medeltiden."
```
Comment (emtalen) [inline cypress/fixtures/articles_index.json]: Why have you added content to the index fixture?

File: cypress/integration/visitorCanReadSpecificArticle.feature.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    });
+  });
+  context("unsuccessfully", () => {
+    beforeEach(() => {
+      cy.server();
+      cy.route({
+        method: "GET",
+        url: "http://localhost:3000/api/v1/articles",
+        response: "fixture:articles_index.json",
+      });
```
Comment (emtalen) [inline cypress/integration/visitorCanReadSpecificArticle.feature.js]: The stubb for the index you can put outside the context block and use for both contexts

File: src/modules/articles.js [hunk 3]
```diff
@@ -1,8 +1,17 @@
...
+  async index() {
+    let result = await axios.get("/articles");
+    return result.data.articles;
+  },
+  async show(articleId) {
+    try {
+      let result = await axios.get(`/articles/${articleId}`);
+      return result.data.article;
+    } catch (error) {
+      return error.response.data.error_message;
```
Comment (emtalen) [inline src/modules/articles.js]: Have you checked this out ugh your api that you get the response in this key? I’m just asking because I’m curious

**Review by emtalen — CHANGES_REQUESTED** [review]

File: cypress/fixtures/articles_index.json (hunk 1 above)
Comment (mauroavellaneda) [reply cypress/fixtures/articles_index.json]: Our mistake, we only added to see how an article looked like in a semantic Card and then we forgot to remove.

File: cypress/integration/visitorCanReadSpecificArticle.feature.js (hunk 2 above)
Comment (mauroavellaneda) [reply cypress/integration/visitorCanReadSpecificArticle.feature.js]: Thanks!!

File: src/modules/articles.js (hunk 3 above)
Comment (mauroavellaneda) [reply src/modules/articles.js]: Thanks to this comment, now it is!

**Pull request merged**